### PR TITLE
Add prctl subcommand PR_GET_DUMPABLE/PR_SET_DUMPABLE

### PR DESCRIPTION
**Pull request by cqs21**
see issue: #960

### CODE REVIEW

**Comment by tatetian** [comment]
@liqinggd Could you review this PR?

File: kernel/aster-nix/src/syscall/prctl.rs [hunk 1]
```diff
@@ -74,8 +87,14 @@ pub enum PrctlCmd {
     PR_GET_NAME(Vaddr),
     PR_SET_TIMERSLACK(u64),
     PR_GET_TIMERSLACK,
+    PR_SET_DUMPABLE(u64),
+    PR_GET_DUMPABLE,
 }
 
+const SUID_DUMP_DISABLE: u64 = 0; /* No setuid dumping */
```
Comment (liqinggd) [inline kernel/aster-nix/src/syscall/prctl.rs:94]: It is better to implement the value `dumpable` as an enum.

**Comment by liqinggd** [comment]
LGTM

**Review by tatetian — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**